[2023-03-01T00:00:16.292+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: save_date_in_file_txt.save_date scheduled__2023-02-28T00:00:00+00:00 [queued]>
[2023-03-01T00:00:16.407+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: save_date_in_file_txt.save_date scheduled__2023-02-28T00:00:00+00:00 [queued]>
[2023-03-01T00:00:16.408+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-01T00:00:16.408+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 4
[2023-03-01T00:00:16.413+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-01T00:00:16.490+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): save_date> on 2023-02-28 00:00:00+00:00
[2023-03-01T00:00:16.505+0000] {standard_task_runner.py:55} INFO - Started process 543 to run task
[2023-03-01T00:00:16.521+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'save_date_in_file_txt', 'save_date', 'scheduled__2023-02-28T00:00:00+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/print_date_file.py', '--cfg-path', '/tmp/tmpvogtwpd3']
[2023-03-01T00:00:16.521+0000] {standard_task_runner.py:83} INFO - Job 28: Subtask save_date
[2023-03-01T00:00:16.720+0000] {task_command.py:388} INFO - Running <TaskInstance: save_date_in_file_txt.save_date scheduled__2023-02-28T00:00:00+00:00 [running]> on host 2b079ebcb28b
[2023-03-01T00:00:16.924+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=save_date_in_file_txt
AIRFLOW_CTX_TASK_ID=save_date
AIRFLOW_CTX_EXECUTION_DATE=2023-02-28T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-02-28T00:00:00+00:00
[2023-03-01T00:00:16.926+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-03-01T00:00:16.928+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'date > ./data']
[2023-03-01T00:00:16.959+0000] {subprocess.py:86} INFO - Output:
[2023-03-01T00:00:16.970+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-03-01T00:00:17.044+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=save_date_in_file_txt, task_id=save_date, execution_date=20230228T000000, start_date=20230301T000016, end_date=20230301T000017
[2023-03-01T00:00:17.117+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-03-01T00:00:17.166+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-03-01T00:45:28.328+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: save_date_in_file_txt.save_date scheduled__2023-02-28T00:00:00+00:00 [queued]>
[2023-03-01T00:45:28.370+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: save_date_in_file_txt.save_date scheduled__2023-02-28T00:00:00+00:00 [queued]>
[2023-03-01T00:45:28.371+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-01T00:45:28.371+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 4
[2023-03-01T00:45:28.372+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-01T00:45:28.406+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): save_date> on 2023-02-28 00:00:00+00:00
[2023-03-01T00:45:28.414+0000] {standard_task_runner.py:55} INFO - Started process 142 to run task
[2023-03-01T00:45:28.420+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'save_date_in_file_txt', 'save_date', 'scheduled__2023-02-28T00:00:00+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/print_date_file.py', '--cfg-path', '/tmp/tmpk7ra13ll']
[2023-03-01T00:45:28.421+0000] {standard_task_runner.py:83} INFO - Job 42: Subtask save_date
[2023-03-01T00:45:28.566+0000] {task_command.py:388} INFO - Running <TaskInstance: save_date_in_file_txt.save_date scheduled__2023-02-28T00:00:00+00:00 [running]> on host 60fce3bd3be2
[2023-03-01T00:45:28.820+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=save_date_in_file_txt
AIRFLOW_CTX_TASK_ID=save_date
AIRFLOW_CTX_EXECUTION_DATE=2023-02-28T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-02-28T00:00:00+00:00
[2023-03-01T00:45:28.823+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-03-01T00:45:28.824+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'date > ./data']
[2023-03-01T00:45:28.851+0000] {subprocess.py:86} INFO - Output:
[2023-03-01T00:45:28.880+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-03-01T00:45:28.948+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=save_date_in_file_txt, task_id=save_date, execution_date=20230228T000000, start_date=20230301T004528, end_date=20230301T004528
[2023-03-01T00:45:29.006+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-03-01T00:45:29.047+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
